root.go

```diff
@@ -25,7 +25,12 @@ import (
 )
 
 var (
-	dbDir = "appdb"
```

Comment:
The changes in this file will somewhat change the usage of docker-bench so @semion-a @jerbia perhaps one of you could comment on whether you have any concerns about that? I think the changes are an improvement e.g.  the test config files are in the cfg directory, so it makes sense to me for that to be the default rather than appdb. (In fact the current code here is boilerplate copied from cobra?)

Reply:
LGTM